assets/scss/modules/_notifications.scss

```diff
@@ -163,7 +163,8 @@
 }
 
 .highlight-message--icon-top {
-  background-position-y: em(20);
+  background-position: em(20) em(20);
```

Comment:
Container padding doesn't affect `background-position` of icon - `top left` positions the icon flush in the corner of the notification area.